src/main/java/no/nav/fo/veilarbarena/scheduled/UserChangePublisher.java

```diff
@@ -96,7 +100,27 @@ public void hentOgPubliserAlleOppfolgingsbrukere() {
                 .executeToList())
                 .map(User::of);
 
-        oppfolgingsbrukere.forEach(this::publish);
+        hentOgleggTilAktoerId(oppfolgingsbrukere)
+                .forEach(this::publish);
+    }
+
+    @Timed(name = "bruker.hent.alle.aktorid")
```

Comment:
Hva er det `@Timed` gjør?

Reply:
Metrikk for å se tiden en metode bruker. Samme som: 
```
 Timer timer = MetricsFactory.createTimer("bruker.hent.alle.aktorid");
            timer.start();
            doStuff();
            timer.stop().report();
```